### ISSUE TITLE
GitHub Pages for spec?

### ISSUE THREAD

**Issue by lsloan** [1EdTech/caliper-spec#458]
**_Update:_**  As mentioned in the comments below, I set up GH Pages on my fork of this repo and the document looks great (as long as it doesn't include any broken links):

* https://lsloan.github.io/caliper-spec/caliper-spec-v1p2.html

It appears that all that's required to make this available from the repo owned by IMSGlobal would be to create a "gh-pages" branch in it and push to that branch the version of the documentation to be published.

---

The new format of the Caliper spec is great, but I miss being able to open the most recent version from the repo and see it as we could when it was written in Markdown.  GitHub won't render HTML pages directly from a repo.  RawGit was a good alternative, but that service shut down last year.

I propose that a [GitHub Pages](https://pages.github.com/) site be set up that could serve up the Caliper spec at a nifty hostname like caliper-spec.github.io or caliper.github.io.  I'm making the assumption that service will allow serving of HTML files.  If that's not correct, maybe there's an alternate way to serve up the documents directly from the repo.

**Comment by arwhyte** [comment]
In the interim you can use https://raw.githack.com/

**Comment by lsloan** [comment]
I didn't know about githack.  Looks like it's taken over from RawGit and expanded the service as well.

* https://raw.githack.com/IMSGlobal/caliper-spec/develop/caliper-spec-v1p2.html

Is that an acceptable solution over GitHub Pages?  It may close down as unceremoniously as did RawGit, but I get the impression GH Pages is probably more stable and wouldn't disappear quickly.

**Comment by lsloan** [comment]
@arwhyte & @ssciolla: I wonder if you'd have time to look at the GitHub Pages I set up on my fork of this repo:

* https://lsloan.github.io/caliper-spec/caliper-spec-v1p2.html

Yesterday during the Caliper weekly meeting, I thought of trying this in my fork.  I only needed to branch the "develop" branch into a new "gh-pages" branch to get it working.  

~~There's a lot of oddness with the table of contents in the document body and the sidebar. It grows horizontally to the right instead of downwards.  I wonder if you have any idea how to fix that.~~

~~I know this isn't a high priority, but I thought it was a good opportunity for me to get acquainted with the mechanics of the spec doc and GH Pages.~~

The problems I saw were resolved by fixing broken links in the document.  See my PR, #459, for the fixes.  When the document was served up by GH Pages, the 404 errors it returned for the broken links included a lot of additional information, including `<meta>` elements which caused page security problems that broke respec's formatting.
